website/server/controllers/api-v3/challenges.js

```diff
@@ -172,7 +289,13 @@ api.joinChallenge = {
  *
  * @apiSuccess {Object} data An empty object
  *
+ * @apiSuccessExample {json} Successfully left challenge
```

Comment:
Hmm.... I don't think it's useful to provide an example for an empty object. Let's just nix these.